openassessment/templates/openassessmentblock/edit/oa_edit_prompt.html

```diff
@@ -20,12 +20,25 @@ <h2>{% trans "Remove" %}</h2>
             <li class="field comp-setting-entry openassessment_prompt_description_wrapper">
                 <div class="wrapper-comp-settings">
                     <textarea class="openassessment_prompt_description setting-input" maxlength="10000" data-w-type="{{w_type}}">{{ prompt_description }}</textarea>
-                    <textarea class="openassessment_prompt_html_content setting-input is--hidden" data-w-type="html-control">{{html_content}}</textarea>
+                    <textarea id="html-content-textarea-{{ counter }}" class="openassessment_prompt_html_content setting-input is--hidden" data-w-type="html-control">{{html_content}}</textarea>
                 </div>
             </li>
             {% if w_type == 'html-control' %}
-            <li class="field comp-setting-entry openassessment_prompt_description_wrapper">
+            <li class="field comp-setting-entry openassessment_prompt_description_wrapper enable-header-editing">
                 {{html_content|safe}}
+                <script type="text/javascript">
```

Comment:
Can we move this inline method to a separate file? Also let's use constants for selectors